packages/app/src/runner/events/capture-protocol.ts

```diff
@@ -1,11 +1,21 @@
 export const addCaptureProtocolListeners = (Cypress: Cypress.Cypress) => {
   Cypress.on('log:added', (log) => {
+    // TODO: UNIFY-1318 - Race condition in unified runner - we should not need this null check
```

Comment:
I added these to mirror: https://github.com/cypress-io/cypress/blob/9d4a3d263a634b4f84daed73a6e3b162d0ad0f60/packages/app/src/runner/event-manager.ts#L497 as I was running into this issue locally.